### PR TITLE
Make HackyAI squad creation a little more flexible and controllable

### PR DESCRIPTION
**Pull request by reaperrr**
This might even be enough to make it somewhat usable for singleplayer missions, as this should allow to artificially limit the attack frequency and squad size.

### CODE REVIEW

File: OpenRA.Mods.Common/AI/HackyAI.cs [hunk 1]
```diff
@@ -43,6 +46,9 @@ public sealed class HackyAIInfo : IBotInfo, ITraitInfo
 		[Desc("Delay (in ticks) between updating squads.")]
 		public readonly int AttackForceInterval = 30;
 
+		[Desc("Minimum delay (in ticks) between creating squads.")]
+		public readonly int MinimumAttackForceDelay = 30;
```
Comment (Mailaender) [inline OpenRA.Mods.Common/AI/HackyAI.cs]: If you set this to zero you avoid a behavior change.

Comment (reaperrr) [inline OpenRA.Mods.Common/AI/HackyAI.cs]: You're right, that would be saver. Will change it.

**Comment by reaperrr** [comment]
Updated.

**Comment by pchote** [comment]
This is a simple extraction of hardcoded constants into parameters.  The defaults are the same, so there will be no ingame changes :+1:.

**Pull request merged**